man/dockerd.8.md

```diff
@@ -165,7 +165,7 @@ $ sudo dockerd --add-runtime runc=runc --add-runtime custom=/usr/local/bin/my-ru
 **-D**, **--debug**=*true*|*false*
   Enable debug mode. Default is false.
 
-**--default-cgroupns-mode**="**host**|**private**"
+**--default-cgroupns-mode**="*host*|*private*"
```

Comment:
Here, both `host` and `private` are literals, i.e. they should be used as-is. Therefore, those are typeset in bold and should not be changed.

OTOH, if that would be something like `default-cgroup-mode=MODE`, the MODE should be in italic, as it's a value to be substituted.